charts/jaeger/Chart.lock

```diff
@@ -3,13 +3,13 @@ dependencies:
   repository: https://charts.helm.sh/incubator
   version: 0.15.3
 - name: elasticsearch
-  repository: https://helm.elastic.co
-  version: 7.17.3
+  repository: https://charts.bitnami.com/bitnami
+  version: 20.0.4
 - name: kafka
   repository: https://charts.bitnami.com/bitnami
-  version: 26.6.2
```

Comment:
Maybe return Kafka to the previous version since it is not related to Elastic search.

Reply:
If all the existing tests work with the new version, then go for it.